README.md

```diff
@@ -1,11 +1,26 @@
 # Kubernetes on CoreOS
 
-Follow the Kubernetes guides on the CoreOS website:
+This repo contains tooling and documention around deploying Kubernetes using CoreOS.
+Initial setup of a Kubernetes cluster is covered, but ongoing maintenance and updates of the cluster is not addressed.
+It is officially maintained by the CoreOS team and meant to be a set of introductory documentation to get a feel for using Kubernetes on CoreOS.
 
-https://coreos.com/kubernetes/docs/latest/
+## The CoreOS Way
+
+When designing these guides and tools, the following considerations are made:
+
+* We always setup TLS
+* An individual node can reboot and the cluster will still function
+* Internal cluster DNS is available
+* Service accounts enabled
+* Use a cloud-provider if we can, for instance on AWS
+* Use Kubernetes for AdmissionControllers and other suggested guidelines
```

Comment:
Nit: Not really sure what this means. Maybe "Use **suggested** Kubernetes AdmissionControllers"?